Super Calculator.py

```diff
@@ -90,7 +90,7 @@ def run(self, edit):
                         if result == '':
                             result = '0'
                     if result != expr:
-                        self.view.replace(edit, region, result)
+                        self.view.replace(edit, region, result if not self.settings.get('keep_expressions') else '{0} {1} {2}'.format(expr, self.settings.get('keep_expressions_separator') or '=', result))
```

Comment:
that's kinda messy implementation you could just use previous if-series to add this logic, or better function-extract refactor that. get provides default value as second argument, format can be explicit with naming, numbered is very old style and kept to be old python compatible (2.6 I believe)

Reply:
I agree even more with you now than before =)
So I just update this PR.

Thanks for helping me improve my code!